src/main/resources/assets/sweet_potato/lang/de_de.json

```diff
@@ -0,0 +1,59 @@
+  "item.sweet_potato.baked_purple_potato": "Gebackene violette Kartoffel",
+  "item.sweet_potato.baked_red_potato": "Gebackene rote Kartoffel",
+  "item.sweet_potato.baked_white_potato": "Gebackene weiße Kartoffel",
+  "item.sweet_potato.potato_powder": "Süßkartoffelpulver",
+  "item.sweet_potato.enchanted_purple_potato": "Verzauberte violette Kartoffel",
+  "item.sweet_potato.enchanted_red_potato": "Verzauberte rote Kartoffel",
+  "item.sweet_potato.enchanted_white_potato": "Verzauberte weiße Kartoffel",
+  "item.sweet_potato.enchanted_wheat_seeds": "Verzauberte Weizenkörner",
+  "item.sweet_potato.enchanted_carrot": "Verzauberte Karotte",
+  "item.sweet_potato.enchanted_potato": "Verzauberte Kartoffel",
+  "item.sweet_potato.enchanted_beetroot_seeds": "Verzauberte Rote-Bete-Samen",
+  "item.sweet_potato.treating_bowl": "Wunschschüssel",
+
+  "block.sweet_potato.purple_potatoes": "Violette Kartoffeln",
+  "block.sweet_potato.red_potatoes": "Rote Kartoffeln",
+  "block.sweet_potato.white_potatoes": "Weiße Kartoffeln",
+  "block.sweet_potato.magic_cube": "Zauberwürfel",
+  "block.sweet_potato.grinder": "Mühle",
+  "block.sweet_potato.agroforestry_table": "Agroforsttisch",
+  "block.sweet_potato.enchanted_acacia_sapling": "Verzauberter Akaziensetzling",
+  "block.sweet_potato.enchanted_dark_oak_sapling": "Verzauberter Schwarzeichensetzling",
+  "block.sweet_potato.enchanted_jungle_sapling": "Verzauberter Tropenbaumsetzling",
+  "block.sweet_potato.enchanted_birch_sapling": "Verzauberter Birkensetzling",
+  "block.sweet_potato.enchanted_oak_sapling": "Verzauberter Eichensetzling",
+  "block.sweet_potato.enchanted_spruce_sapling": "Verzauberter Fichtensetzling",
+  "block.sweet_potato.enchanted_sugar_cane": "Verzaubertes Zuckerrohr",
+  "block.sweet_potato.enchanted_acacia_leaves": "Verzaubertes Akazienlaub",
+  "block.sweet_potato.enchanted_birch_leaves": "Verzaubertes Birkenlaub",
+  "block.sweet_potato.enchanted_dark_oak_leaves": "Verzaubertes Schwarzeichenlaub",
+  "block.sweet_potato.enchanted_jungle_leaves": "Verzaubertes Tropenbaumlaub",
+  "block.sweet_potato.enchanted_oak_leaves": "Verzaubertes Eichenlaub",
+  "block.sweet_potato.enchanted_spruce_leaves": "Verzauberte Fichtennadeln",
+  "block.sweet_potato.enchanted_wheat": "Verzauberter Weizen",
+  "block.sweet_potato.enchanted_carrots": "Verzauberte Karotten",
+  "block.sweet_potato.enchanted_potatoes": "Verzauberte Kartoffeln",
+  "block.sweet_potato.enchanted_beetroots": "Verzauberte Rote Bete",
+
+  "subtitles.block.sweet_potato.agroforestry_table.finish": "Nutzpflanze aufgewertet",
+
+  "advancement.husbandry.sweet_potato.get_sweet_potato.title": "Süß und weich",
+  "advancement.husbandry.sweet_potato.get_sweet_potato.description": "Süßkartoffel erhalten",
+  "advancement.nether.sweet_potato.get_magic_cube.title": "Magie der Seele",
+  "advancement.nether.sweet_potato.get_magic_cube.description": "Zauberwürfel erhalten",
+
+  "container.sweet_potato.magic_cube": "Zauberwürfel",
```

Comment:
Magic Cube's translation has been changed to Magischer Topf.

Reply:
Alright!